install-panel.sh

```diff
@@ -604,9 +604,21 @@ function debian_based_letsencrypt {
   systemctl stop nginx
 
   echo -e "\nMake sure you choose Option 1, and create a Standalone Web Server during the certificate"
-  certbot certonly -d "$FQDN"
+  certbot certonly --standalone -d "$FQDN"
 
-  systemctl restart nginx
+  if [ -d "/etc/letsencrypt/live/$FQDN/" ]; then
+    systemctl restart nginx
+  else
+    print_warning "Obtaining a certificate failed."
```

Comment:
```suggestion
    print_warning "The process of obtaining a Let's Encrypt certificate failed!"
```

Perhaps something like this? Language police here, sorry .... :slightly_smiling_face: